src/test/kotlin/org/rust/ide/lineMarkers/RsCrateDocLineMarkerProviderTest.kt

```diff
@@ -3,11 +3,10 @@
  * found in the LICENSE file.
  */
 
-package org.rust.ide.annotator
+package org.rust.ide.lineMarkers
 
 import org.rust.ProjectDescriptor
 import org.rust.WithStdlibAndDependencyRustProjectDescriptor
-import org.rust.ide.lineMarkers.RsLineMarkerProviderTestBase
```

Comment:
Let's avoid unrelated changes in the same commit. It makes future problem investigation harder. I'd extract this change into a separate commit